.jenkins/scripts/kafka-rocksdbjni-s390x.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/docker-images/kafka-based/kafka/Makefile b/docker-images/kafka-based/kafka/Makefile
+index d5fde4588..c23f9112d 100644
+--- a/docker-images/kafka-based/kafka/Makefile
++++ b/docker-images/kafka-based/kafka/Makefile
+@@ -42,6 +42,8 @@ clean:
+ .kafka-$(KAFKA_VERSION).tmp: ../../artifacts/binaries/kafka/archives/kafka_2.13-$(KAFKA_VERSION).tgz
+ 	test -d tmp/$(KAFKA_VERSION) || mkdir -p tmp/$(KAFKA_VERSION)
+ 	tar xvfz ../../artifacts/binaries/kafka/archives/kafka_2.13-$(KAFKA_VERSION).tgz -C tmp/$(KAFKA_VERSION) --strip-components=1
++	test -f tmp/$(KAFKA_VERSION)/libs/rocksdbjni-6.22.1.1.jar && cp -f $(HOME)/$(ROCKSDBJNI_DIR)/rocksdbjni-6.22.1.1.jar tmp/$(KAFKA_VERSION)/libs/ || true
++	test -f tmp/$(KAFKA_VERSION)/libs/rocksdbjni-6.19.3.jar && cp -f $(HOME)/$(ROCKSDBJNI_DIR)/rocksdbjni-6.19.3.jar tmp/$(KAFKA_VERSION)/libs/ || true
+ 	touch .kafka-$(KAFKA_VERSION).tmp
+ 
+ .kafka-$(KAFKA_VERSION)-ignorelist.tmp: ../../artifacts/binaries/kafka/$(KAFKA_VERSION).ignorelist
```

Comment:
I'm not sure I follow what does this do / why we need it. My understanding is that RocksDB is used by Kafka Streams only => so it is not used in anyway in brokers, Connect, etc. So why do we care about the version? Also, this will not get into the released images. So I'm not sure we want it in the images which will ne under test. Any reliance on this would ultimately fail when running the pipelines against the release images build outside of the Jenkins environment.

Reply:
Thanks @scholzj ! Since RocksDB is not directly used by Kafka Operator and this patch will not get into the released image, I removed the patch from this PR. So far we haven't seen any system tests affected by it.